### DISCUSSION TITLE
Trojan/malware in deta.exe

### DISCUSSION THREAD

**Discussion by javigaralva** [deta/cloud-docs#607] [General]
Hi, I've found a Trojan/malware (Trojan.Malware.300983.susgen) in the deta executable. 
Here is the report of virustotal:

https://www.virustotal.com/gui/file/8bcec75f57603316eb213b4c84addeedd1cc528c3702604534c0c2d3cbbdc4c3?nocache=1

**Comment by etrotta** [comment]
1 security vendor and no sandboxes flagged this file as malicious, out of 72 vendors. I feel like it's pretty safe to assume that it is just a false positive.
Furthermore, if you google that "Trojan.Malware.300983.susgen", you'll see a lot of examples of other software (unrelated to deta) being flagged with it as well.

**Reply by AyushSehrawat** [reply]
Its false probs, maybe it just meant potential harmful :thonk:..

**Reply by AyushSehrawat** [reply]
Still, @abdelhai can you look into it if you get some time :)

**Comment by javigaralva** [comment]
Thanks for your replies :-)

**Reply by el-garro** [reply]
On virustotal, pay attention to the big vendors like KAV or ESET. Some smaller vendors have the reputation to detect almost anything and ask for a "fee" to whitelist the programs. I'm not accusing MaxSecure of being that kind of shady business though.

**Comment by rohanshiva** [comment]
It should just be a false positive. Seems to be a [common issue](https://go.dev/doc/faq#virus) for some Go distributions/binaries.
